activerecord/test/cases/transaction_callbacks_test.rb

```diff
@@ -689,12 +691,12 @@ class TopicWithCallbacksWithSpecificOrderWithSettingFalse < ActiveRecord::Base
   def test_callbacks_run_in_order_defined_in_model_if_using_run_after_transaction_callbacks_in_order_defined
     topic = TopicWithCallbacksWithSpecificOrderWithSettingTrue.new
     topic.save
-    assert_equal [1, 2, 3, 4, "save", "create"], topic.history
+    assert_equal [1, 2, "prepend 2", "prepend 1", 3, 4, "save", "create"], topic.history
```

Comment:
Failure before the patch and with the new callbacks added above:

<img width="1174" alt="Screenshot 2024-03-04 at 11 21 30 AM" src="https://github.com/rails/rails/assets/54629302/a9ea352d-5a20-49ae-b32d-c2b0a2d8e8b9">